app/controllers/api/v1/slack/user_info_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Api
+  module V1
+    module Slack
+      class UserInfoController < ApplicationController
+        before_action VerifyRequest 
+ 
+        # get all users 
+        def index
+          render json: Users.all
```

Comment:
We probably do not need to send over every attribute on `User`.  We could instead update the [user serializer](https://github.com/OperationCode/operationcode_backend/blob/master/app/serializers/user_serializer.rb) and repurpose that.

Reply:
Would the call be, I changed the serialization in the file but I'm not sure how to reference it. I thought that `Users.all` would use the default serialization. 
`render json: Users.attributes`